### DISCUSSION TITLE
n1盒子无法安装，出错如下，请问是什么原因，谢谢

### DISCUSSION THREAD

**Discussion by hmm1204** [messense/aliyundrive-webdav#60] [Q&A] [answered]
```bash
root@OpenWrt:~# opkg install aliyundrive-webdav_0.4.1-1_aarch64_generic.ipk
Unknown package 'aliyundrive-webdav'.
Collected errors:
 * pkg_hash_fetch_best_installation_candidate: Packages for aliyundrive-webdav found, but incompatible with the architectures configured
 * opkg_install_cmd: Cannot install package aliyundrive-webdav.
```

**Comment by messense** [comment]
`opkg print-architecture` 输出贴一下看看。

**Reply by hmm1204** [reply]
```bash
root@OpenWrt:~# opkg print-architecture
arch all 1
arch noarch 1
arch aarch64_cortex-a53 10
```

**Reply by wufuhai** [reply]
这才是万能的正解，只不过作者没说明，一般人看不懂。用opkg print-architecture命令查看系统的架构然后找到对应版本架构的程序按照就行了

**Comment by messense** [comment, marked as answer]
应该下载安装 aarch64_cortex-a53 架构版本的主程序，比如 https://github.com/messense/aliyundrive-webdav/releases/download/v0.4.2/aliyundrive-webdav_0.4.2-1_aarch64_cortex-a53.ipk

**Reply by ohjust** [reply]
碰到同样的问题，N1,小钢炮下面winscap下安装，/tmp$ opkg install aliyundrive-webdav_1.0.0_aarch64_cortex-a53.ipk
Collected errors:
 * opkg_solver_install: Cannot install package aliyundrive-webdav.
Not selecting aliyundrive-webdav 1.0.0 due to incompatible architecture.
Unknown package 'aliyundrive-webdav'.

**Comment by lawzhougc** [comment]
遇到一样的情况  最后安装了aarch64_generic版本成功

**Comment by fengyuclub** [comment]
wget https://github.com/messense/aliyundrive-webdav/releases/download/v2.3.2/aliyundrive-webdav_2.3.2-1_aarch64_generic.ipk
wget https://github.com/messense/aliyundrive-webdav/releases/download/v2.3.2/luci-app-aliyundrive-webdav_2.3.2_all.ipk
wget https://github.com/messense/aliyundrive-webdav/releases/download/v2.3.2/luci-i18n-aliyundrive-webdav-zh-cn_2.3.2-1_all.ipk
opkg install *.ipk --force-reinstall

**Reply by ldqrecord** [reply]
太强了
